src/output/plugins/snapcast/SnapcastOutputPlugin.cxx

```diff
@@ -103,7 +103,7 @@ SnapcastOutput::Unbind() noexcept
 inline void
 SnapcastOutput::AddClient(UniqueSocketDescriptor fd) noexcept
 {
-	auto *client = new SnapcastClient(*this, std::move(fd));
+	auto client = std::make_unique<SnapcastClient>(*this, std::move(fd));
```

Comment:
Same use-after-free crash bug as the one you added to `HttpdOutput::AddClient()`.